assets/i18n/en.json

```diff
@@ -120,11 +120,13 @@
     "I18N_DELETE_ACCOUNT_PAGE_BUTTON": "Delete My Account",
     "I18N_DELETE_ACCOUNT_PAGE_HEADING": "Delete Account",
     "I18N_DELETE_ACCOUNT_PAGE_LIST_1_1": "User and email settings",
-    "I18N_DELETE_ACCOUNT_PAGE_LIST_1_2": "User statistics",
-    "I18N_DELETE_ACCOUNT_PAGE_LIST_1_3": "Private explorations, collections, topics, stories, skills and questions",
+    "I18N_DELETE_ACCOUNT_PAGE_LIST_1_2": "User statistics that are not useful for analysis",
```

Comment:
I think this is a bit confusing, can you clarify it? In particular -- analysis by who?

What are examples that go in the first category vs the second? Feel free to ping me and I can help you with the language.

Reply:
I went through it again and realized taht we keep all the statistical data, so removed it.